breaking-front/src/constants/path.js

```diff
@@ -9,6 +9,7 @@ export const PAGE_PATH = {
   PROFILE_EDIT: '/profile/edit',
   POST: (postId) => `/post/${postId}`,
   POST_WRITE: '/post/write',
+  POST_EDIT: '/post/edit',
```

Comment:
```suggestion
  POST_EDIT: (postId) => `/post/${postId}/edit`,
```
이렇게 하는 건 어떨까요? 만약 유저가 작성하지 않은 페이지면 에러페이지를 띄우거나 alert를 띄워 못들어가게 하면 좋을 것 같습니다.

Reply:
1. 프로필 수정페이지와 같게 post/edit으로 통일했습니다.
2. post_edit에 postId가 url에 필요한 이유를 찾지 못했습니다.